joomla/joomla-cms
# [4.0] ftp boolean error in com_ joomlaupdate

Pull Request for Issue # .

### Summary of Changes
There is a error when ftp is enabled when you run joomla update you get a warning if debug is enabled.
![image](https://user-images.githubusercontent.com/1850089/73938447-e4e4dc00-48ac-11ea-9bc3-84a825bded0b.png)

### Testing Instructions
Turn debug on.
Set reporting level to development.
check joomla update component.
warning should be displayed.
apply pr.
warning should go away.


### Expected result
![image](https://user-images.githubusercontent.com/1850089/73938622-44db8280-48ad-11ea-859a-b1c05f99f1e6.png)



### Actual result
![image](https://user-images.githubusercontent.com/1850089/73938447-e4e4dc00-48ac-11ea-9bc3-84a825bded0b.png)


### Documentation Changes Required
none
